pkg/agent/packetcapture/capture/bpf_test.go

```diff
@@ -115,7 +116,39 @@ func TestCalculateInstructionsSize(t *testing.T) {
 			count:     15,
 			direction: crdv1alpha1.CaptureDirectionSourceToDestination,
 		},
-
+		{
+			name: "proto with port and flags",
```

Comment:
I feel like the test name should be more precise and indicate which flags are matching